extras/tie-breaker/tie-breaker.yaml

```diff
@@ -0,0 +1,20 @@
+# -*- mode: yaml -*-
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: tie-breaker-pod
+  labels:
+    app: gluster-tie-breaker
+spec:
+  containers:
+  - name: tie-breaker
+    image: docker.io/kadalu/gluster-tie-breaker:latest
```

Comment:
name mentioned above is "kadalu-tiebreaker-app" (https://github.com/kadalu/kadalu/pull/168/files#diff-bacb243d4cac3bc29dd35ad5d1ac8469R16)

Reply:
not sure what I was thinking. Good catch! Fixed in the update.